Resources/Prototypes/Entities/Objects/Weapons/Guns/Projectiles/projectiles.yml

```
@@ -1088,26 +1088,26 @@
     forceSound: true
 
 - type: entity
-  name: wide laser barrage
+  name: laser barrage
   id: BulletLaserSpread
   parent: BulletLaser
   categories: [ HideSpawnMenu ]
```

YAMLlint (1.35.1)

[error] too many spaces inside brackets

(brackets)